apps/site/pages/en/learn/asynchronous-work/discover-promises-in-nodejs.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Discover Promises in Node.js
+layout: learn
+authors: avivkeller
+---
+
+# Discover Promises in Node.js
+
+A **Promise** is a special object in JavaScript that represents the eventual completion (or failure) of an asynchronous operation and its resulting value. Think of a Promise as a placeholder for a value that is not yet available but will be in the future.
+
+Think of a Promise like ordering a pizza: you don't get it right away, but the delivery person promises to bring it to you later. You don't know _exactly_ when, but you know the outcome will either be "pizza delivered" or "something went wrong."
+
+## Promise States
+
+A Promise can be in one of three states:
+
+- **Pending**: The initial state, where the asynchronous operation is still running.
+- **Fulfilled**: The operation completed successfully, and the Promise is now resolved with a value.
+- **Rejected**: The operation failed, and the Promise is settled with a reason (usually an error).
+
+For example, imagine you're waiting for an email from a friend. You're in the pending state. If your friend replies, you've entered the fulfilled state. If the email bounces back, you are in the rejected state.
```

Comment:
You use a pizza delivery example above then switch to an email example here. Likely better to just continue the pizza delivery theme.... When you order the pizza you're in the `Pending` state. When the pizza arrives you're in the `Fulfilled` state. If the Pizza delivery scooter breaks down on the way to your house and the pizza tragically falls to the ground, you're in the `Rejected` state... etc.